skietbaan_fe/src/components/Login.js

```diff
@@ -17,9 +17,19 @@ class Login extends Component {
     this.Login = this.Login.bind(this);
     this.handleChange = this.handleChange.bind(this);
     this.Validate = this.Validate.bind(this);
-
+    this.togglePassword = this.togglePassword.bind(this);
+    this.ToggleNavbar = this.ToggleNavbar.bind(this);
```

Comment:
Keep consistency with your naming.
this.ToggleNavbar should be this.toggleNavBar

Reply:
fixed